src/tests/main-window.test.ts

```diff
@@ -89,22 +91,24 @@ describe( 'fullscreen events', () => {
 	} );
 
 	it( 'sends fullscreen-change event when entering fullscreen', () => {
-		const mockSend = jest.fn();
-		createdWindow.webContents.send = mockSend;
-
 		// Simulate entering fullscreen
 		createdWindow.emit( 'enter-full-screen' );
 
-		expect( mockSend ).toHaveBeenCalledWith( 'window-fullscreen-change', true );
+		expect( sendIpcEventToRendererWithWindow ).toHaveBeenCalledWith(
+			createdWindow,
+			'window-fullscreen-change',
+			true
+		);
```

Comment:
Now that we have the `sendIpcEventToRendererWithWindow` abstraction, it's more fitting to mock that function instead of the underlying `BrowserWindow.webContents.send`. I hoped this would potentially fix the flaky tests, but it did not.